### PR TITLE
Split server test cases into multiple files.

### PR DESCRIPTION
**Pull request by gimite**
#62

### CODE REVIEW

File: tests/server_test_cases/api_key_management_tests.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+import reveal
+import scrape
+import setup_pf as setup
+from test_pfif import text_diff
+from text_query import TextQuery
+import utils
+from server_tests_base import ServerTestsBase
+
+
+# TODO(ryok): fix go_as_operator() and re-enable the tests.
```
Comment (skywhale) [inline tests/server_test_cases/api_key_management_tests.py]: Oops. I really should find time to fix this...

**Comment by skywhale** [comment]
Thank you!! This was really a long waited CL, and I'm very happy that I finally landed. We might want to clean up the imports a bit, but I don't feel strongly about it.

**Comment by skywhale** [comment]
Could you resolve the conflicts and merge?

**Comment by skywhale** [comment]
LGTM

**Comment by gimite** [comment]
Thanks for the review.

**Pull request merged**